### PR TITLE
Member page will not stuck on loading if response is null (Not part of any organization)

### LINKED ISSUE #429: The member page stuck if not joined into any organization.(Bug)
Issue Description: The member page stucks if not joined into any organization.
Issue Severity: medium
Environment Details: Issue found mostly in the talawa\lib\views\pages\organization\members
Observed Behaviour: The member page stucks if not joined into any organization.
Expected Behaviour: clicking yes should allow us to join the organization.
Does this issue need immediate attention?  yes
Are you willing to work on this issue: Yes, I am willing to work on this issue. Please assign this to me!
Capture
.dart

### PR DESCRIPTION
**Pull request by yasharth291**
**What kind of change does this PR introduce?**

[BUG] it fixes a bug

**Did you add tests for your changes?**

no

**Summary**

the motivation for this bugfix is that when the user was not joined to any organization the member page was stuck on circular loading. So this motivated me to fix this bug.

**Does this PR introduce a breaking change?**

no

**Other information**

Fixes #429 

https://user-images.githubusercontent.com/55004621/111903678-7d1a3200-8a69-11eb-81c0-e344fb57a5fe.mp4

### CODE REVIEW

File: lib/views/pages/members/members.dart [hunk 1]
```diff
@@ -22,13 +24,15 @@ class Organizations extends StatefulWidget {
...
+
+  String currentOrgID;
   List alphaMembersList = [];
   int isSelected = 0;
   Preferences preferences = Preferences();
 
 
   //providing initial states to the variables
-  initState() {
+   initState( )  {
```
Comment (Sagar2366) [inline lib/views/pages/members/members.dart:35]: Sorry, I did not understand requirement for this change ?

Comment (yasharth291) [reply lib/views/pages/members/members.dart:35]: no sir there is no need of this it is just by mistake i clicked on spacebar to look spatial

**Comment by Sagar2366** [comment]
@yasharth291 add unit tests for your PR to merge it.

**Comment by yasharth291** [comment]
okay sir

**Comment by Sagar2366** [comment]
@yasharth291 pull latest master changes for travis ci issue fix and send PR again for review.

**Comment by yasharth291** [comment]
> @yasharth291 pull latest master changes for travis ci issue fix and send PR again for review.

@Sagar2366 done sir

**Comment by Sagar2366** [comment]
> > @yasharth291 pull latest master changes for travis ci issue fix and send PR again for review.
> 
> @Sagar2366 done sir

Add unit tests/widget tests for your changes.

**Comment by yasharth291** [comment]
> > > @yasharth291 pull latest master changes for travis ci issue fix and send PR again for review.
> > 
> > 
> > @Sagar2366 done sir
> 
> Add unit tests/widget tests for your changes.

ok sir doing it right now

**Comment by yasharth291** [comment]
@Sagar2366  the test are added sir

**Comment by Sagar2366** [comment]
@yasharth291 travis ci is still failing, fix it

**Comment by yasharth291** [comment]
> @yasharth291 travis ci is still failing, fix it

checking it sir

**Comment by yasharth291** [comment]
travis ci is now working @Sagar2366

**Review by Sagar2366 — APPROVED** [review]

**Pull request merged**